app/services/articles/feeds/large_forem_experimental.rb

```diff
@@ -105,7 +105,9 @@ def score_followed_organization(article)
       end
 
       def score_experience_level(article)
-        - (((article.experience_level_rating - (@user&.experience_level || 5)).abs / 2) * @experience_level_weight)
+        # rubocop:disable Layout/LineLength
```

Comment:
Instead of ignoring the cop, we could do the following which ends up taking one line less in total:

```ruby
user_experience_level = @user&.setting&.experience_level || 5
- (((article.experience_level_rating - user_experience_level).abs / 2) * @experience_level_weight)
```